packages/p2p/src/components/my-profile/my-profile-stats/my-profile-stats-table/my-profile-stats-table.scss

```diff
@@ -1,9 +1,13 @@
 .my-profile-stats-table {
     border-bottom: none;
     grid-template-columns: repeat(4, 1fr);
-    margin: 2.4rem 0;
+    margin: 1.4rem 0;
     overflow: auto;
 
+    &.dc-table__row {
+        height: unset;
```

Comment:
Reduce height to `0%` since due to `my-profile__content` is now `flex`, the stats are stretched:
![image](https://user-images.githubusercontent.com/103016120/182793655-d2a43bb1-bb63-4789-8c95-a1c88b24a237.png)

### Before
![Screenshot 2022-08-04 at 3 51 36 PM](https://user-images.githubusercontent.com/103016120/182793806-0e1ae588-f39c-42c3-844d-3c8f9e652f7b.png)


### After
<img width="1112" alt="Screenshot 2022-08-04 at 3 52 14 PM" src="https://user-images.githubusercontent.com/103016120/182793899-a54ac7d4-882b-4f55-8bef-0c3679251383.png">